lang/python/python/Makefile

```diff
@@ -175,9 +175,10 @@ define Build/Compile/python-setuptools
 		--cache-dir "$(DL_DIR)/pip-cache" \
 		install \
 		--ignore-installed \
-		--root=$(PKG_BUILD_DIR)/install-setuptools --prefix=. \
+		--root=$(PKG_BUILD_DIR)/install-setuptools \
+		--prefix=/usr \
```

Comment:
i also thought about changing this a few times;
i guess i started doing it, then gave up midway;

Reply:
Seeing `--prefix=.` was a little too scary for me :laughing: